### PR TITLE
Migration to lightbend repo

### PR DESCRIPTION
**Pull request by 2m**
Fixing the fallout after #509

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ There isn't a schema language or anything like that. However, two
 suggested tools are:
 
  - use the
-   [checkValid() method](http://typesafehub.github.io/config/latest/api/com/typesafe/config/Config.html#checkValid-com.typesafe.config.Config-java.lang.String...-)
+   [checkValid() method](http://lghtbend.github.io/config/latest/api/com/typesafe/config/Config.html#checkValid-com.typesafe.config.Config-java.lang.String...-)
```
Comment (marcospereira) [inline README.md]: `lghtbend`.

Comment (2m) [reply README.md]: Thanks for catching.

**Comment by dwijnand** [comment]
There's a lot more than 26 instances:

```
$ git show-ref HEAD
08f5a6020ea0770440529ff9d0e2191c72367bec refs/remotes/origin/HEAD
$ git grep typesafehub | wc -l
      65
```

**Comment by 2m** [comment]
Thanks for noticing. IntelliJ did not replace all of the occurences. And there is something strange with this PR where it does not show the new commits I pushed to the branch. I will reopen this PR.

**Pull request closed without merging**